faiss/gpu/test/test_gpu_index.py

```diff
@@ -219,11 +219,9 @@ def test_stress(self):
 
 
 class TestShardedFlat(unittest.TestCase):
-
+    
+    unittest.skipIf(faiss.get_num_gpus() == 1,"relevant for multiple GPU only")
```

Comment:
Since `unittest.skipIf()` is a decorator, you need to call it as `@unittest.skipIf()`.

Reply:
@beauby my bad updated :)